SYCL/ESIMD/dpas/dpas_test1.cpp

```diff
@@ -0,0 +1,75 @@
+//==--------------- dpas_test1.cpp  - DPC++ ESIMD on-device test -----------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu-intel-dg2
```

Comment:
As we talked, there are no DG2 or PVC machines in CI, so this will disable even compilation. Are you going to address this in a separate commit?

Reply:
No, per discussion with CI folks, compilation tests are supposed to be checked in to compiler tests, while llvm-test-suite contains executable tests. So, it looks like CI would need to be extended to have these machines.

Reply:
I see, thanks